lightning/src/blinded_path/message.rs

```diff
@@ -30,6 +32,8 @@ pub(crate) struct ReceiveTlvs {
 	/// sending to. This is useful for receivers to check that said blinded path is being used in
 	/// the right context.
 	pub(crate) path_id: Option<[u8; 32]>,
+	/// Custom Tlvs
+	pub(crate) custom_tlvs: Vec<(u64, Vec<u8>)>,
```

Comment:
> @valentinewallace is this one also LDK-specific in the way payment::ReceiveTlvs is?

Yep!

Reply:
I was planning to handle the changes related to OnionMessage, and BlindedPayment seperately.
But it seems more logical now to handle them together.
Introduced custom_tlvs support for `payment::ReceiveTlvs` in c9d57f8345b4ec0a54a57718abcae59d11ccd343
Thanks!

Reply:
> @valentinewallace is this one also LDK-specific in the way `payment::ReceiveTlvs` is?
> 
> In any case, we should really focus on including custom tlvs in `payment::ReceiveTlvs` (or, not sure if we need "generic custom TLVs" so much as "user provides one `Vec<u8>` that we put in one TLV).

@TheBlueMatt FYI, the BOLT 12 payment notifications scheme that Jeff wrote up includes blinded payment paths that aren't constructed by the receiver, so this may not be LDK specific anymore.

Reply:
Updated the custom tlvs to be simple Vec<u8> in [pr2830.03](https://github.com/shaavan/rust-lightning/commits/pr2830.03)
Thanks for the suggestion!